emailApp/dataCleaning/cleaner.js

```diff
@@ -0,0 +1,48 @@
+// checks the email validation
+function validEmail(e) {
+  var filter = /^\s*[\w\-\+_]+(\.[\w\-\+_]+)*\@[\w\-\+_]+\.[\w\-\+_]+(\.[\w\-\+_]+)*\s*$/;
+  return String(e).search(filter) != -1;
+}
+
+function fillArray(myString) {
```

Comment:
Please add one-line description about the function.

Reply:
done